src/components/experience/Experience.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import './Experience.scss';
+
+const experienceData = [
```

Comment:
@sadanandpai I will create mockdata folder and add over there for now

Reply:
sure